plugin/trino-loki/src/test/java/io/trino/plugin/loki/LokiQueryRunner.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.loki;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+import io.airlift.log.Level;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.github.jeschkies.loki.LokiTestServer;
```

Comment:
The `LokiTestServer` class should be located in this package. It's using `compose.yaml`, which is not referenced explicitly anywhere in this PR. Finding https://github.com/jeschkies/loki-client-java/blob/main/loki-client-testutils/src/main/java/io/github/jeschkies/loki/LokiTestServer.java might be hard/annoying to find. 

I understand you did that to make this PR smaller, but it's hiding the complexity, not removing it. It'll make it harder to support this plugin after it gets merged.

Reply:
>  It's using compose.yaml, which is not referenced explicitly anywhere in this PR.

I actually wanted to move the compose into the test utils of the client. Would that be ok for? Or should I move the test server back?

Reply:
It makes tests harder to follow. What's the value of moving it to a separate library?

Reply:
The idea was to reduce the lines to review. The server is really small though. I brought it back.